### ISSUE TITLE
Is test.py deprecated?

### ISSUE THREAD

**Issue by MedievalMatt** [loris-imageserver/loris#406]
Hello,

I'm switching over my hosting and reinstalling Loris on the new environment. I noticed that the current version at /loris no longer has test.py, but the installation instructions still point to it as a necessary step. Moreover, when I copied test.py over to my new environment and ran it prior to setup the installation worked, but setup.py alone would not.  Is it intentional that test.py is no longer in the files available for download?

**Comment by alexwlchan** [comment]
Which part of the installation instructions are you reading that mention `test.py`? I can't seem to find what you're looking at.

**Comment by MedievalMatt** [comment]
Hi Alex,

Thanks for responding.  Going back and looking at the instructions it looks like the issue is not on the github directions directly, but on the page people are directed to if they’re having problems on Debian.  

All best,
—Matt

**Comment by alexwlchan** [comment]
Right, thanks. We should probably add a note to reflect the fact that it’s out-of-date then.
